config.yml

```diff
@@ -177,22 +171,18 @@ genesis:
       supplierList:
         - address: pokt19a3t4yunp0dlpfjrp7qwnzwlrzd5fzs2gjaaaj
           services:
-            - endpoints:
+            - service:
+                id: anvil
```

Comment:
Only referencing the `Service.Id` makes sense and also raises the question about embedding full entities into others (the whole `Service` is embedded into the `Supplier` in this case) .

It pushes devs to use the embedded data instead of querying the source of truth.

You already solved one of these which was about the service's `ComputeUnitsPerRelay` where the dev used the value embedded in the `Application.Services.ComputeUnitsPerRelay` instead of fetching the `Service` which is the source of truth.

We should (at some point) only embed ids and enforce fetching the rest of the data from the SOT.

Reply:
Updated some TODOs. PTAL

This also made me realize that onchain data structures are like micro-services.